sdk/helper/storagepacker/ids2keys.go

```diff
@@ -0,0 +1,99 @@
+package storagepacker
+
+import (
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"github.com/hashicorp/vault/sdk/helper/cryptoutil"
+	"math"
+	"sort"
+	"strings"
+)
+
+type itemRequest struct {
+	// ID is the Item ID, provided by client
+	ID string
+
+	// Key is the identifier used internally, a hash of ID
+	Key string
+
+	// Value is the stored object, nil if not found
+	Value *Item
+}
+
+func GetItemIDHash(itemID string) string {
+	return hex.EncodeToString(cryptoutil.Blake2b256Hash(itemID))
+}
+
+// Length of key
+const KeyLength = 64
+
+// keysForIDs calculates keys and generates itemRequests for each input ID.
+func (s *StoragePackerV2) keysForIDs(ids []string) []*itemRequest {
```

Comment:
Why use keys everywhere when the functions deal with requests? e.g. why not `requestsForIDs` here and `requestsForItems` below and naming the file `ids2requests.go`?

Also given that not all functions in here are dealing with converting ids to requests, perhaps a better name would be something like `util.go`.

Reply:
Renamed these methods, split the file into keys.go and items.go.